src/fuzzer/Fuzzer.ts

```diff
@@ -380,7 +380,7 @@ const _checkStopCondition = (
   }
 
   // End testing if we exceed the maximum number of duplicates generated
-  if (currentDupeCount >= Math.max(env.options.maxTests, 1000)) {
+  if (currentDupeCount >= env.options.maxDupeInputs) {
```

Comment:
Yes,`maxDupeInputs` is the maximum duplicate inputs generated _in a row_ rather than globally. Agree this should be made clear in the (i) tab and in the More Options... UI also---e.g., maybe we try putting it on its own row down at the bottom of the tab since it's expected to be uncommon for folks to want to mess with it?)